### PR TITLE
add AWS example

### PR DESCRIPTION
**Pull request by zreigz**
Add AWS example

### CODE REVIEW

File: examples/aws/main.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      aws {
+        access_key_id     = "set_it_up"
+        secret_access_key = "set_it_up"
+      }
+    }
+  }
+}
+resource "kubermatic_node_deployment" "example_node" {
+  name       = "examplenode"
+  cluster_id = kubermatic_cluster.example_cluster.id
```
Comment (mfranczy) [inline examples/aws/main.tf:23]: I think this example won't pass, as I saw the cluster_id format changed
https://github.com/kubermatic/terraform-provider-kubermatic/blob/master/kubermatic/resource_node_deployment.go#L31

**Review by mfranczy — CHANGES_REQUESTED** [review]

File: examples/aws/main.tf (hunk 1 above)
Comment (zreigz) [reply examples/aws/main.tf:23]: I've tested it. Works fine

Comment (mfranczy) [reply examples/aws/main.tf:23]: ok

**Review by mfranczy — APPROVED** [review]
/lgtm

**Pull request merged**